src/app/templates/app/base.html

```diff
@@ -181,13 +182,15 @@
     </div><!--/.nav-collapse -->
   </nav>
 
-    <div class="container body1">
```

Comment:
Removing `container body1` div will cause changes on other pages.

We don't need to create separate block for dots. The frame will be visible below the spdx navigation bar.

Reply:
I'm not much of an HTML person, but I did go through all the pages with this pull request and didn't see any difference in the rendering.

Even though this change could impact other pages, it doesn't look like it actually does - so I'm OK merging as is.

@rtgdk - agree?

Reply:
Yeah I agree. Merging this